app/src/main/java/com/dessalines/rankmyfavs/ui/components/common/Dialogs.kt

```diff
@@ -172,3 +182,43 @@ fun PreviewAreYouSureDialog() {
         onYes = {},
     )
 }
+
+@Composable
+fun ColorPickerDialog(
+    onColorSelected: (Color) -> Unit,
+    onDismissRequest: () -> Unit,
+    controller: ColorPickerController,
+) {
+    Dialog(onDismissRequest = onDismissRequest) {
+        Surface(
+            shape = RoundedCornerShape(12.dp),
+            color = MaterialTheme.colorScheme.surface,
+            modifier = Modifier.padding(16.dp),
```

Comment:
Externalize these paddings and corner sizes ( I think these are in Sizes.kt) , or just use one of the existing MEDIUM_PADDING or whatever, which I'm pretty sure is already here.